packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -162,11 +165,13 @@ class DatePickerInput extends React.Component {
         />
         <Button
           className={styles.button}
+          text={buttonText}
           onClick={this.handleOnButtonClick}
           onKeyDown={this.handleOnKeyDown}
           icon={Icon}
+          isIconOnly
           isCompact
-          type="button"
+          isDisabled={additionalInputProps.disabled}
```

Comment:
Should we make `isDisabled` a first-class prop?

Reply:
Made `disabled` a first class prop in terra-date-picker, terra-date-time-picker and terra-time-input for consistency.
https://github.com/cerner/terra-core/pull/1206/commits/bca7c3a5dc1fc9c6e61dee6c5d2b52f13ea9eca8